internal/ls/format.go

```diff
@@ -123,3 +126,55 @@ func (l *LanguageService) getFormattingEditsAfterKeystroke(
 	}
 	return nil
 }
+
+// Unlike the TS implementation, this function *will not* compute default values for
+// `precedingToken` and `tokenAtPosition`.
+// It is the caller's responsibility to call `astnav.GetTokenAtPosition` to compute a default `tokenAtPosition`,
+// or `astnav.FindPrecedingToken` to compute a default `precedingToken`.
+func getRangeOfEnclosingComment(
```

Comment:
This function in Strada used to differentiate between a `null` `precedingToken` (which would be left undefined) and an `undefined` `precedingToken` (which would get replaced with a default computed value). We can't easily do this in Go, so, I'm simply not doing anything to the parameters here.

Reply:
I kinda dislike the original behavior, and there was only one place which needed it in Strada, so I'd rather leave it like this.